src/readerrdy.coffee

```diff
@@ -136,8 +136,12 @@ class ConnectionRdyState extends NodeState
       Enter: ->
         @raise 'bump'
       bump: ->
-        if @connRdy.availableRdy <= @connRdy.lastRdySent * 0.25
-          @connRdy.setRdy @connRdy.maxConnRdy
+        # No need to keep setting the RDY count for versions of NSQD after
+        # 0.3.0.
```

Comment:
Versions after 0.3.0 or version 0.3.0 and after? Seems pedantic, but this comment implies the check below should be `version <= [0, 3, 0]`